src/pyhf/workspace.py

```diff
@@ -770,7 +773,7 @@ def combine(cls, left, right, join='none', merge_channels=False):
             'observations': new_observations,
             'version': new_version,
         }
-        return cls(newspec)
+        return cls(newspec, validate=validate)
```

Comment:
This should get covered by a test. I can add that though unless someone beats me to it.